examples/misc/pythonic_matplotlib.py

```diff
@@ -3,9 +3,9 @@
 Pythonic Matplotlib
 ===================
 
-Some people prefer to write more pythonic, object-oriented code
-rather than use the pyplot interface to matplotlib.  This example shows
-you how.
+Some people prefer to write more "Pythonic", explicitly object-oriented code
+rather than use the implicit pyplot interface to Matplotlib.  This example
+shows you how.
```

Comment:
```suggestion
rather than use the implicit pyplot interface to Matplotlib. This example
shows you how to take advantage of the explicit Matplotlib interface.
```